test/views/agent/partials/VatReturnsPartialSpec.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.agent.partials
+
+import assets.messages.partials.{VatReturnsPartialMessages => Messages}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.mvc.AnyContentAsEmpty
+import play.api.test.FakeRequest
+import views.ViewBaseSpec
+
+class VatReturnsPartialSpec extends ViewBaseSpec {
+
+  "VatReturnsPartial" should {
+
+    "passed a mandation status of 'Non MTDfB'" should {
+
+      lazy implicit val testGetRequest: FakeRequest[AnyContentAsEmpty.type] = FakeRequest("GET", "")
+      lazy val view = views.html.agent.partials.vatReturnsPartial("Non MTDfB")(messages, mockConfig)
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      s"have the correct card heading" in {
+        elementText(".heading-medium") shouldBe Messages.heading
+      }
+
+      "display the body of text for the card" in {
+        elementText("#card-info") shouldBe Messages.paragraphOneNonMandated
```

Comment:
Instead of selecting the id, you can use `elementText("p")`. If there are multiple paragraphs on the page and you want to select a specific one, you can use `elementText("p:nth-of-type(1)")`